docs/tutorials/google/calibration_api.ipynb

```diff
@@ -0,0 +1,609 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "S97D7gQHMdG_"
+   },
+   "outputs": [],
+   "source": [
+    "##### Copyright 2021 The Cirq Developers"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellView": "form",
+    "id": "Y6Wh1qtGMghL"
+   },
+   "outputs": [],
+   "source": [
```

Comment:
this inspired me to finally open https://github.com/quantumlib/Cirq/issues/4260

Reply:
+1 to that.